marketplace/deployer_helm_tiller_base/test/schema.yaml

```diff
@@ -0,0 +1,34 @@
+properties:
+application_api_version: v1beta1
+properties:
+  name:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+  application_uid:
+    type: string
+    x-google-marketplace:
+      type: APPLICATION_UID
+  marketplace-k8s-app-tools-helm-hooks.image:
+    type: string
+    x-google-marketplace:
+      type: IMAGE
```

Comment:
Can we have a special property for the deployer image? I think the hook should just use the deployer image.

Reply:
Maybe this isn't the right use of the deployer image, and we should bake/publish a different one?

Reply:
The issue with having a different image is that the partner has to copy that image into their staging repo. It's not terrible for our partners, but it doesn't seem necessary.